server/image_status.go

```diff
@@ -35,11 +34,12 @@ func (s *Server) ImageStatus(ctx context.Context, req *types.ImageStatusRequest)
 		return nil, err
 	}
 	var (
+		status   *pkgstorage.ImageResult
 		notfound bool
 		lastErr  error
 	)
 	for _, image := range images {
-		status, err := s.StorageImageServer().ImageStatus(s.config.SystemContext, image)
+		status_, err := s.StorageImageServer().ImageStatus(s.config.SystemContext, image)
```

Comment:
I would expect us to not need the additional variable. is it the case that `ImageStatus` could return both an error and a non-nil value?

Reply:
The `status_` variable is removed in a later commit; it is used here because `status, err :=` would, IIRC, not update the `status` variable from outside the loop.

Alternatively, this could be

```go
var err error
status, err = …
```

In a later commit, both `status` and `err` become local to the loop body, and this inelegance disappears.

Reply:
WFM, I was going commit by commit